packages/analytics/src/helpers.test.ts

```diff
@@ -213,7 +214,11 @@ describe('Gtag wrapping functions', () => {
       expect((window['dataLayer'] as DataLayer).length).to.equal(1);
     });
 
-    it('new window.gtag function does not wait when sending "set" calls', async () => {
+    it('new window.gtag function does not wait when sending "consent" calls', async () => {
```

Comment:
Sorry if I'm just misreading this because of the Github diff UI but is this replacing the "set" test? Should it be a copy?